tests/suites/test_suite_cipher.arc4.data

```diff
@@ -42,11 +42,11 @@ ARC4 Encrypt and decrypt 31 bytes
 depends_on:MBEDTLS_ARC4_C
 enc_dec_buf:MBEDTLS_CIPHER_ARC4_128:"ARC4-128":128:31:-1
 
-ARC4 Encrypt and decrypt 32 bytes
+ARC4 Encrypt and decrypt 32 bytes [#1]
 depends_on:MBEDTLS_ARC4_C
 enc_dec_buf:MBEDTLS_CIPHER_ARC4_128:"ARC4-128":128:32:-1
 
-ARC4 Encrypt and decrypt 32 bytes
+ARC4 Encrypt and decrypt 32 bytes [#2]
```

Comment:
Wrong description - 33 bytes. Following - please remove the `[#1]` from the test above and `[#2]` from this test.

Reply:
Rejected as out of scope.